libs/agno/agno/embedder/huggingface.py

```diff
@@ -36,16 +34,21 @@ def client(self) -> InferenceClient:
         return self.huggingface_client
 
     def _response(self, text: str):
-        return self.client.post(json={"inputs": text}, model=self.id, task=self.task)
+        return self.client.feature_extraction(text=text, model=self.id)
```

Comment:
Would this require the latest version of their library to work?

Reply:
i did not upgrade the library but it worked, so ig not